app/views/admin/products/show.html.erb

```diff
@@ -9,7 +9,11 @@
         <%= t "show_product_admin.image" %>
       </th>
       <th class="image">
-        <%= gravatar_for @product %>
+        <% if @product.image.attached? %>
+          <%= image_tag @product.display_image %>
+        <% else %>
+          <%= gravatar_for @product %>
+        <% end %>
```

Comment:
code này lặp nè
tạo 1 method để dùng chung nhé

Reply:
dạ

Reply:
cái này e xin dùng riêng , vì 1 cái em nhận @product, 1 cái e dùng trong for each là product .